legacy/application/models/StreamSetting.php

```diff
@@ -55,16 +55,26 @@ public static function getOutput($key, $add_prefix = false)
                 $prefix . 'pass' => $output['source_password'] ?? '',
                 $prefix . 'admin_user' => $output['admin_user'] ?? 'admin',
                 $prefix . 'admin_pass' => $output['admin_password'] ?? '',
-                $prefix . 'channels' => $output['audio']['channels'] ?? 'stereo',
-                $prefix . 'bitrate' => $output['audio']['bitrate'] ?? 128,
-                $prefix . 'type' => $output['audio']['format'],
                 $prefix . 'name' => $output['name'] ?? '',
                 $prefix . 'description' => $output['description'] ?? '',
                 $prefix . 'genre' => $output['genre'] ?? '',
                 $prefix . 'url' => $output['website'] ?? '',
                 $prefix . 'mobile' => $output['mobile'] ?? 'false',
                 // $prefix . 'liquidsoap_error' => 'waiting',
             ];
+            if (array_key_exists('audio', $output)) {
+                $result = array_merge($result, [
+                    $prefix . 'channels' => $output['audio']['channels'] ?? 'stereo',
+                    $prefix . 'bitrate' => $output['audio']['bitrate'] ?? 128,
+                    $prefix . 'type' => $output['audio']['format'],
+                ]);
+            } elseif ($output['kind'] == 'hls') {
+                // HLS : set web server host and port
```

Comment:
here we could hack the mount point to add 'hls/' prefix instead of mandate it in configfile mount setting....I haven't do it because it's a bit hacky

Reply:
We want good default, but they should be configurable. Having docs/a comment say 'you need hls/ as prefix' is not optimal.

Reply:
i generate mount with a prefix hls now